### ISSUE TITLE
Add balanced accuracy for imbalanced classification cases

### ISSUE THREAD

**Issue by ljchang** [ljchang/nltools#36]

**Comment by ljchang** [comment]
There are two ways to do this.  One randomly select cases from each class until balanced.  Two, use the weighted class functionality in sklearn.  This is already possible by passing argument through kwargs.  Might be helpful to make this more explicit with documentation in the predict function though.
